domains/harshmallick.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "My portfolio website",
+   "repo": "https://github.com/harsh-mallick/portfolio",
+   "owner": {
+     "username": "harsh-mallick",
+     "email": "harshmallick052009@gmail.com"
+   },
+   "record": {
+     "URL": "harshmallick.vercel.app"
```

URL records require protocol
```suggestion
     "URL": "https://harshmallick.vercel.app/"
```